pyproject.toml

```diff
@@ -47,6 +47,7 @@ beautifulsoup4 = "^4.12.3"
 pytest-repeat = "^0.9.3"
 flask = "^3.1.0"
 python-dateutil = "^2.9.0.post0"
+tox = "^4.23.2"
```

Comment:
What's this? Looks like we removed <kbd>tox</kbd> from dependencies above on L41 of Dockerfile.desktop and don't see it in the new list of dependencies.

Reply:
Tox through APT is old, so I have to install the python package